backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc

```diff
@@ -41,4 +41,11 @@ include::modules/oadp-installing-dpa-1-2-and-earlier.adoc[leveloffset=+1]
 include::modules/oadp-installing-dpa-1-3.adoc[leveloffset=+1]
 include::modules/oadp-enabling-csi-dpa.adoc[leveloffset=+2]
 
+.Additional resources
```

Comment:
This is a global comment because I see this in a few places, but the .Additional resources sections should only contain text inside the xref:...[only text here]. In Customer Portal, the other text doesn't render properly. See this for more info: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections

Reply:
Also, the "see" construction for a list doesn't follow IBMSG recommendations for lead-in wording: https://www.ibm.com/docs/en/ibm-style?topic=format-lists#lead-in-wording